ani-cli

```diff
@@ -613,21 +641,10 @@ if [ -z "$select_first" ]; then
 			ep_choice_end=""
 			;;
 		a)
-			tput reset
-			prompt "Search Anime"
-			query="$REPLY $REPLY2"
-			search_results=$(search_anime "$query")
-			[ -z "$search_results" ] && die "No search results found"
-			anime_selection "$search_results"
-			episode_selection
+            search_another_anime
```

Comment:
Making these functions elsewhere in the script is a welcome change, as it makes our interactive part more readable.
Good job here

Reply:
:+1: